pages/hugo.yml

```diff
@@ -50,9 +50,9 @@ jobs:
         run: "[[ -f package-lock.json || -f npm-shrinkwrap.json ]] && npm ci || true"
       - name: Build with Hugo
         env:
-          # For maximum backward compatibility with Hugo modules
+          HUGO_CACHEDIR: ${{ runner.temp }}/hugo_cache
           HUGO_ENVIRONMENT: production
-          HUGO_ENV: production
+          TZ: America/Los_Angeles
```

Comment:
That does not sound like a sensible default that would work for everyone.

Reply:
If you have a more sensible default that would be great. The reason to include TZ is to demonstrate how to change it.

Reply:
Hugo's configuration file is the place that would make sense: https://gohugo.io/getting-started/configuration/#timezone. I.e. in the repository along with the static content instead of in the workflow ;)

So I would just remove that line from the workflow.

Reply:
Thank you for your review. I've removed the TZ environment variable from the pull request. We'll keep the TZ environment variable in our [documented workflow example](https://gohugo.io/hosting-and-deployment/hosting-on-github/#procedure) because Hugo's `timeZone` configuration setting is only applicable to date values without timezone offsets.